client/src/components/layout/HelForm.jsx

```diff
@@ -102,13 +102,14 @@ const HelForm = ({ submitForm, submitted, user }) => {
   };
 
   const handleCourseAddition = e => {
-    if (!humanitiesCourses.includes(e.target.innerHTML)) {
+    let event = (e.target.innerHTML).toLowerCase();
+    if (!humanitiesCourses.includes()) {
       if (isEditingRowAtIndex !== -1) {
```

Comment:
This thing displays in the list after "lowercasing" it. 
I was hoping it should be displayed exactly as the user sees it in the catalog. 
Only change to lowercase while saving to the database